### ISSUE TITLE
Problem with XA30 DICOM conversion detecting corrupt slice times for CMRR data

### ISSUE THREAD

**Issue by michaelhallquist** [rordenlab/dcm2niix#762]
**Describe the bug**

Our PRISMA 3T scanner was recently upgraded to XA30. We run CMRR multiband BOLD sequences with sub-second temporal resolution. These files are now throwing errors in dcm2niix conversion, whereas the old Siemens software/DICOMs did not. I have provided an example DICOM. I looked at the DICOM in pydicom and found that the RepetitionTime is 635.0, which makes sense, and the json shows .635 as the TR (correct).

But the dcm2niix (current Apr 11 version) fails with the complaint in the log below about bogus or corrupt slice times.

**To reproduce**

1. Run the command 'dcm2niix .'
2. See error below

**Expected behavior**

In the past, the slice timing information would be extracted into the JSON.

**Output log**

Chris Rorden's dcm2niiX version v1.0.20230411  (JP2:OpenJPEG) (JP-LS:CharLS) GCC8.5.0 x86-64 (64-bit Linux)
Found 1211 DICOM file(s)
Slices not stacked: dimensions vary across slices
Warning: Siemens MoCo? Bogus slice timing (range -1..-1, TR=635 seconds)
Convert 1210 DICOM as ./_RIDL2_100144_8 (82x82x54x1210)
Warning: Slice timing appears corrupted (range 0..0, TR=635 ms)
Convert 1 DICOM as ./_RIDL2_100144_8a (82x82x18x1)
Conversion required 12.590864 seconds (2.019853 for core code).

**Version**

Please report the complete version string:

Chris Rorden's dcm2niiX version v1.0.20230411  Clang14.0.3 x86-64 (64-bit MacOS)
v1.0.20230411

**Troubleshooting**

I have tried the latest release and the development release, but I get the same result.

Thanks for any help you can offer. It's possible the sequence is somehow setup improperly, but this is affecting multiple groups at our site who run different variants of the CMRR multiband BOLD sequences. All are having this problem.

[1.3.12.2.250209.195112.249631.991321.601758.313825.MR.dcm.zip](https://github.com/rordenlab/dcm2niix/files/12862033/1.3.12.2.250209.195112.249631.991321.601758.313825.MR.dcm.zip)

**Comment by neurolabusc** [comment]
@michaelhallquist I am keen to resolve your issue as our center will also upgrade our VE11 Prisma Fit to XA. What distiguishes your data from other [XA30 data I have seen](https://github.com/neurolabusc/dcm_qa_xa30) is that it lacks the private tag [TimeAfterStart (0021,1104)](https://github.com/rordenlab/dcm2niix/issues/303). Your images **do** include FrameAcquisitionDateTime (0018,9074) but dcm2niix explicitly ignores this with the [note](https://github.com/rordenlab/dcm2niix/blob/fe2f26005109f396a4f828aa438241f73fc25fe8/console/nii_dicom.cpp#L5682C16-L5682C25):

```
in theory, 0018,9074 could provide XA10 slice time information, but scrambled by XA10 de-identification: better to use 0021,1104
```

Can I suggest you work with your Siemens Research Collaboration Manager and the CMRR team to understand why the TimeAfterStart is not being populated.

**Comment by michaelhallquist** [comment]
Thanks very much for looking at this, @neurolabusc! I have shared your feedback with the MR physicist working on this problem and he will initiate a conversation with CMRR and Siemens to sort it out. I'll report back on what we learn!

**Comment by michaelhallquist** [comment]
Hi Chris, I'm still waiting on a final diagnosis from our MR center. But in the interest of keeping you in the loop, @neurolabusc, I wanted to mention that it appears that the DICOMs lost the TimeAfterStart tags during the data transfer to Flywheel. Our university uses this service for synchronizing and hosting DICOMs. Yet, I've learned that Flywheel has anonymization procedures that modify DICOMs, potentially in deleterious ways. We processed XA30 data for the same subject from a DVD cut directly on the Siemens console computer and did not have major issues. There were a few small, ignorable warnings, but slice times, TE, and TRs all came through as expected in the JSON.

We are working to figure out how to modify the DICOM edits within Flywheel and I can report back if that is of interest.
